src/jit/hwintrinsiclistxarch.h

```diff
@@ -22,6 +22,13 @@
     8) Each intrinsic has one category with type of `enum HWIntrinsicCategory`, please see the definition of HWIntrinsicCategory for details
     9) Each intrinsic has one or more flags with type of `enum HWIntrinsicFlag`
 */
+// ***************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************
+//                 Intrinsic ID                                           Function name                                   ISA         ival        SIMD size       NumArg                                                                                                     instructions                                                                                                     Category                            Flags
+//                                                                                                                                                                             {TYP_BYTE,              TYP_UBYTE,          TYP_SHORT,          TYP_USHORT,         TYP_INT,            TYP_UINT,           TYP_LONG,           TYP_ULONG,          TYP_FLOAT,          TYP_DOUBLE}
+// ***************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************
+//  Helper Intrinsics
+HARDWARE_INTRINSIC(System_Runtime_Intrinsics_Vector128_Zero,               "get_Zero",                                    Base,         -1,              16,           0,     {INS_invalid,           INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid},           HW_Category_Helper,                 HW_Flag_NoContainment|HW_Flag_NoRMWSemantics)
```

Comment:
Is the shorter name `Vector128_Zero` not enough?

Reply:
Fixed to be `Base_Vector128_Zero` (given that we have prefixed the "ISA" on the other intrinsics as well)